timesteps: 12000
model_type: 'sac'
model_path: '/models/checkpoints/sac_checkpoint_19_03_2024_15_34_8300_steps' # set either path from bimanual_handover package or empty string
checkpoint: True # If starting point is checkpoint set True, otherwise False

# Env paramters
contact_modality: 'effort' # effort or tactile
env_check: True
record: False

observation_space:
    joint_values: False
    pca: True
    effort: True
    tactile: False
    one_hot: True

# Only used for new models
architecture:
    entropy_coefficient: 0.001
    # List of layer sizes for actor network
    actor: 
        - 256
        - 256
    # List of layer sizes for critic network
    critic:
        - 256
        - 256
